src/CoreShop/Component/Index/Getter/FieldCollectionGetter.php

```diff
@@ -45,7 +51,19 @@ public function get(IndexableInterface $object, IndexColumnInterface $config): m
 
         foreach ($validItems as $item) {
             if (method_exists($item, $fieldGetter)) {
-                $fieldValues[] = $item->$fieldGetter();
+                if (!empty((new \ReflectionMethod($item, $fieldGetter))->getParameters())) {
+
+                    $fallbackMemory = DataObject\Localizedfield::getGetFallbackValues();
```

Comment:
CoreShop already has a Helper for inheritance to make that better readable and re-usable, we should introduce that for Localized Fallbacks as well. (see https://github.com/coreshop/CoreShop/blob/master/src/CoreShop/Component/Pimcore/DataObject/InheritanceHelper.php for more).

Reply:
didn't know about it. Tnx